features/settings/pubspec.yaml

```diff
@@ -16,6 +16,8 @@ dependencies:
     path: ../../core
   domain:
     path: ../../domain
+  navigation:
```

Comment:
add all navigation dependencies to core and add export of it in core.dart file

Reply:
Dont get it